cloudformation/media-atom-maker.json

```diff
@@ -357,7 +338,9 @@
                                 "aws.dynamo.auditTableName=", { "Ref": "AuditTable" }, "\n",
                                 "EOF\n",
 
-                                "systemctl start media-atom-maker\n"
+                                "aws s3 cp s3://atom-maker-dist/media-service/", {"Ref":"Stage"}, "/media-atom-maker/media-atom-maker.deb /tmp",
```

Comment:
I'm not that happy with current structure of the dist folder - feels like we have two too many folders. I won't change it now but I'm happy to sort this out a later date. cc/ @akash1810 @mbarton @Reettaphant